equinix/README.md

```diff
@@ -40,29 +45,29 @@ The `harvester.install.config_url=https://metadata.platformequinix.com/userdata`
 
     ```yaml
     #cloud-config
-    token: token  # replace with a desired token
+    token: token           # replace with a desired token
     os:
       ssh_authorized_keys:
-      - ssh-rsa ...  # replace with your public key
-      password: p@ssword  # replace with a your password
+      - ssh-rsa ...        # replace with your public key
+      password: p@ssword   # replace with a your password
+      ntp_servers:
+      - 0.suse.pool.ntp.org
+      - 1.suse.pool.ntp.org
     install:
       mode: create
-      networks:
-        harvester-mgmt: # The management bond name. This is mandatory.
-          interfaces:
-          - name: eth0
-          default_route: true
-          method: dhcp
       device: /dev/sda
       iso_url: https://releases.rancher.com/harvester/master/harvester-master-amd64.iso
       tty: ttyS1,115200n8
+      vip: <server_public_ip> # Randomly fill in the IP first or reinstall when the Public IP is known
```

Comment:
@ibrokethecloud is this the best method we can do for now of the Equinix platform installation?

Reply:
@guangbochen we can create an elastic IP in advance and attach it to the instance, and then use the same in vip.

This is what my sample terraform module does.